src/unix/Asai_unix.ml

```diff
@@ -1,82 +1,115 @@
 open Bwd
 
-module StringTbl = Hashtbl.Make(String)
-
 open Asai
 
 open Notty
 open Notty.Infix
 
-
-module Make (ErrorCode : ErrorCode.S) =
+module Make (Code : Code.S) =
 struct
-  module Diagnostic = Diagnostic.Make(ErrorCode)
-
   let vline ~attr height str =
     I.vcat @@ List.init height (fun _ -> I.string ~attr str)
+  let column ~align images =
```

Comment:
I don't think we need to compute the width for line numbers here, shouldn't `<|>` and  `vcat` work?

Reply:
We need it for right aligned line numbers

Reply:
Ah, make sense!